log4j-docgen/pom.xml

```diff
@@ -27,6 +27,7 @@
   <artifactId>log4j-docgen</artifactId>
 
   <properties>
+    <maven.compiler.release>17</maven.compiler.release>
```

Comment:
The `log4j-plugins` artifact is a **required** dependency (since I am actually **using** the annotation interfaces in the code), so the bytecode levels must match.